features/admin-file-attributes.feature

```diff
@@ -0,0 +1,142 @@
+@admin-file @issue-457
+  Feature: A curator can manage file attributes in admin file update page
+    As a curator,
+    I want to manage file attributes from the update form
+    So that I can associate various attributes to files
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+    And an admin user exists
+
+  @ok
+  Scenario: Guest user cannot visit admin file update page
+    Given I am not logged in to Gigadb web site
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
+
+  @ok
+  Scenario: Go to a published dataset found in production-like database
+    Given I am not logged in to Gigadb web site
+    When I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    Then I should see "History" tab with text "File Termitomyces_assembly_v1.0.fa.gz updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @published
+  Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons
```

Comment:
Your `Scenario` descriptions could be shortened and therefore faster to read. For example, you don't need to say `Sign in as Admin` because it is already in the `Given` line immediately below. Therefore the `Scenario` could be `See Attribute, Edit, Delete, Save buttons on admin file update page`.